src/download.py

```diff
@@ -19,6 +19,9 @@
     "speakers?questions=all",
 ]
 
+if not Config.raw_path.exists():
```

Comment:
You could use `Config.raw_path.mkdir(parents=True, exist_ok=True)` ([docs](https://docs.python.org/3/library/pathlib.html#pathlib.Path.mkdir)).

Doesn't really matter, feel free to leave it as it is :)

Reply:
Thanks :heart: